Symfony/CS/Tokenizer/Transformer/ArrayTypehint.php

```diff
@@ -28,8 +28,11 @@ public function process(Tokens $tokens)
     {
         foreach ($tokens->findGivenKind(T_ARRAY) as $index => $token) {
             $nextIndex = $tokens->getNextMeaningfulToken($index);
-            $nextToken = $tokens[$nextIndex];
+            if (null === $nextIndex) {
+                return;
```

Comment:
The PSR0 fixer generates `<?php class array` and ask the tokenizer for tokens of that code, then this will break down.

Can be tested by adding;
```php
            array(
                '<?php class array',
                array()
            ),
```
on line https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/Symfony/CS/Tests/Tokenizer/Transformer/ArrayTypehintTest.php#L38